src/status_im/utils/prices.cljs

```diff
@@ -10,14 +11,17 @@
 ;; Example usage:
 ;; (get-prices "ETH" "USD" println print)
 
-(def api-url "https://min-api.cryptocompare.com/data")
+(defn api-url [chaos-mode?]
+  (if chaos-mode?
+    "https://min-api.thisurldoesntexist5b60647d927da46.com/data"
```

Comment:
same in case we want it to be up but return a given status code `http://httpstat.us/500`